jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -30,11 +30,168 @@
 /** \addtogroup ecma ECMA
  * @{
  *
- * \addtogroup ecmatypedarrayobject ECMA %TypedArray% object related routines
+ * \addtogroup ecmatypedarrayobject ECMA TypedArray object related routines
  * @{
  */
 
 /**
+ * set typedarray's element value
+ *
+ * @return ecma_number_t: the value of the element
+ */
+static ecma_number_t
+get_typedarray_element (lit_utf8_byte_t *src, /**< the location in the internal arraybuffer */
+                        lit_magic_string_id_t class_id) /**< class name of the typedarray */
+{
+#define GET_ELEMENT(type, location, number) \
```

Comment:
Place of `GET_ELEMENT` and `SET_ELEMENT` macros are a bit odd to me. I'd move them out of the functions add descriptions to them too.